docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,254 @@ grand_parent: "General"
+
+#### Including the Pre-packaged Version
+
+For the pre-packaged version, we should include the bpmn-js viewer via HTML script tag:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-viewer.production.min.js"></script>
+```
+
+or for the navigated viewer:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-navigated-viewer.production.min.js"></script>
+```
+
+#### Including via NPM
+
+First, we should install bpmn-js via NPM:
+```
+npm install bpmn-js
+```
+
+We should then import the BPMN viewer as an ES module:
+
+```javascript
+import Viewer from 'bpmn-js/lib/Viewer';
+```
+
+or for the navigated viewer:
+
+```javascript
+import NavigatedViewer from 'bpmn-js/lib/NavigatedViewer';
+```
+
+#### Viewing the BPMN 2.0 XML
+
+Now that we're done with the setup, we should construct a new bpmn-js Viewer instance. If we imported the bpmn-js as pre-packaged, we should use the `BpmnJS` constructor:
+
+```javascript
+const viewer = new BpmnJS({ container: '#canvas' });
+```
+
+if we've imported the bpmn-js via NPM, we should go for the `Viewer` constructor instead:
+```javascript
+const viewer = new Viewer({ container: '#canvas' });
+```
+
+or for the navigated viewer:
+```javascript
+const viewer = new NavigatedViewer({ container: '#canvas' });
+```
+
+Note that `#canvas` here refers to the [root DOM element that we've created in our HTML file](#including-the-root-element).
+
+We can then import a BPMN 2.0 XML using the [importXML API](/general/api-reference/bpmn-js/importXML). For this demo, we can use this simple BPMN file:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_0xcv3nk" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="3.0.0">
+  <bpmn:process id="Process_0sckl64" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1" />
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_0sckl64">
+      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
+        <dc:Bounds x="179" y="159" width="36" height="36" />
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</bpmn:definitions>
+```
+
+We can directly set this to a variable or we can store this file somewhere and fetch it. For simplicity, we won't deal with this here:
```

Comment:
I don't think we need to explain how to get hold of an XML string. We should assume that users are able to do this.

Reply:
Well to be honest we've seen some trivial stuff asked to us in the forum, so we can't always assume users know about stuff like this. I think it does not hurt anyone to keep this remark here. If you have a strong opinion on this one we can remove it, but again I see no point in removing. We even have this [url viewer example](https://github.com/bpmn-io/bpmn-js-examples/tree/master/url-viewer) demonstrating one can fetch the XML instead of hardcoding it.